packages/ui-app/src/styles/media.css

```diff
@@ -0,0 +1,43 @@
+/* Copyright 2017-2019 @polkadot/ui-app authors & contributors
+/* This software may be modified and distributed under the terms
+/* of the Apache-2.0 license. See the LICENSE file for details. */
+
+.ui--media-large,
+.ui--media-medium,
+.ui--media-small {
```

Comment:
Your naming convention for class names is different in most other files:
The first letter after the `--` should be uppercase: `.{appname}--{Specific-class}`

Reply:
Not for ui— (eg. ui—row,grid,...)

Those are “globals” that apply anywhere.